sources/packages/backend/apps/queue-consumers/test/helpers/testing-modules/testing-modules-helper.ts

```diff
@@ -89,6 +89,6 @@ export async function createTestingAppModule(): Promise<CreateTestingModuleResul
  */
 function createClamAVServiceMock(): ClamAVService {
   const mockedClamAVService = {} as ClamAVService;
-  mockedClamAVService.scanFile = jest.fn(() => Promise.resolve(false));
+  mockedClamAVService.scanFile = jest.fn(() => Promise.resolve(null));
```

Comment:
Why was it changed to `null`?
I believe keeping it as `false` was to ensure the happy path scenario. 
Any E2E that needs a different value should change it.